components/layout/navigation.tsx

```diff
@@ -14,8 +14,8 @@ export function Navigation() {
 
   const menuClasses = (isActive: boolean) => {
     return [
-      "inline-flex flex-col items-center justify-center h-12 px-4 rounded-md",
-      isActive ? "text-blue-600 font-semibold" : "text-gray-600",
+      "inline-flex flex-col items-center justify-center h-12 px-2 rounded-md",
+      isActive ? "text-blue-600 font-medium" : "text-gray-600",
```

Comment:
```suggestion
      isActive ? "text-blue-600 font-semibold" : "text-gray-600",
```

The design on the Figma file uses a `font-size` of 600, which is `font-semibold`. Please keep it as it is.

![image](https://user-images.githubusercontent.com/5663877/126999077-1c29b25e-f779-46d1-9446-4c52c872084e.png)